run-tests.sh

```diff
@@ -251,6 +251,8 @@ runTestsIfServiceVersion "@indexRecordConsentCodes" "sheepdog" "1.1.13"
 runTestsIfServiceVersion "@coreMetadataPage" "portal" "2.20.8"
 runTestsIfServiceVersion "@indexing" "portal" "2.26.0" "2020.05"
 runTestsIfServiceVersion "@cleverSafe" "fence" "4.22.4" "2020.09"
+runTestsIfServiceVersion "@requestor" "requestor" "1.5.0" 
```

Comment:
did you test removing requestor from the manifest and deployments, to see if the tests are disabled as expected?

Reply:
I actually I didn't

Reply:
I tried both with arborist and requestor versions, it is working as expected.